src/Catel.Fody/Weaving/AutoProperties/OnPropertyChangedWeaver.cs

```diff
@@ -67,26 +67,29 @@ private bool AddOrUpdateOnPropertyChangedMethod(PropertyDefinition property)
 
                     foreach (var propertyDefinition in dependentProperties)
                     {
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldarg_1));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Callvirt, getMethodReference));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldstr, property.Name));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Callvirt, stringEqualsMethodReference));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldc_I4_0));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ceq));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Stloc_0));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldloc_0));
-
-                        var jmpIdx = idx++;
-                        onPropertyChangedMethod.Body.Instructions.Insert(jmpIdx, Instruction.Create(OpCodes.Nop));
-
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Nop));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldarg_0));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Ldstr, propertyDefinition.Name));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Call, _catelType.RaisePropertyChangedInvoker));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Nop));
-                        onPropertyChangedMethod.Body.Instructions.Insert(idx++, Instruction.Create(OpCodes.Nop));
-
-                        onPropertyChangedMethod.Body.Instructions[jmpIdx] = Instruction.Create(OpCodes.Brtrue_S, onPropertyChangedMethod.Body.Instructions[idx - 1]);
+                        if (!_catelType.NoWeavingProperties.Contains(propertyDefinition))
```

Comment:
I have a habit to avoid instructions like `continue`,`break`, more than one `return` in a method. Probably is a "wrong" habit, but is just about semantics, so, better reading. 

For instance, if a `foreach` have a `break` instruction for a condition, such construction negates the `foreach` semantic. So, typically if I have to write down such `foreach` I turn it into a `while`. 

Isn't the same case, but (for me) the `continue` approach adds some complexity to read the method because adding a "non-interesting condition just for a jump to the end of the loop with `continue`" makes me read the jump ignore condition first and then read the rest of the method. 

Probably isn't the best approach (for runtime and compiler optimizations) but is better for reading ;).

I will change it anyway.